credentials/utopia-dl/queries.json

```diff
@@ -20,5 +20,34 @@
         "ecdsa-sd-2023"
       ]
     }
+  },
+  "Driver's License (Name & Number Only)": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your ISO 18013 Drivers License Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://www.w3.org/ns/credentials/examples/v2",
+          "https://w3id.org/vdl/v1",
+          "https://w3id.org/vdl/aamva/v1"
```

Comment:
I wonder if the aamva one is needed for this query or not. This is something we'll want to clean up over time -- ensuring we only request the needed contexts and that the wallet only delivers those requested.

Reply:
I don't think we need any AAMVA defined terms in the example.

Reply:
Updated: aea7f8c